prow/config.yaml

```diff
@@ -16268,6 +16268,11 @@ periodics:
       - |-
         --query=is:pr
         -label:do-not-merge
+        -label:do-not-merge/blocked-paths
```

Comment:
nit: possible to use a regex here instead?

Reply:
AFAIK, no.

Reply:
@stevekuznetsov what do you think about creating a do-not-merge pluging that adds/removes the `do-not-merge` label based on the presence/absence of any `do-not-merge/REASON` labels?

Reply:
Do we know that specifying more labels in this list increases the complexity of the query and expends more tokens? I think having that kind of circuitous behavior may be confusing, especially if we still want to allow humans to add `do-not-merge` and we would need to list events to see if we can remove the label.

Reply:
There can be multiple reasons that a PR cannot merge: it may update paths that are blocked, and it may lack a release note. I don't think we want to muck with them.